functional-tests/test-bed/src/test/java/org/arquillian/smart/testing/ftest/configuration/SkipTestExecutionFunctionalTest.java

```diff
@@ -0,0 +1,99 @@
+package org.arquillian.smart.testing.ftest.configuration;
+
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+import java.util.Collection;
+import net.jcip.annotations.NotThreadSafe;
+import org.arquillian.smart.testing.ftest.testbed.project.Project;
+import org.arquillian.smart.testing.ftest.testbed.rules.GitClone;
+import org.arquillian.smart.testing.ftest.testbed.rules.TestBed;
+import org.arquillian.smart.testing.ftest.testbed.testresults.TestResult;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+
+import static org.arquillian.smart.testing.ftest.testbed.configuration.Mode.ORDERING;
+import static org.arquillian.smart.testing.ftest.testbed.configuration.Strategy.AFFECTED;
+import static org.assertj.core.api.Assertions.assertThat;
+
+@Category(NotThreadSafe.class)
+public class SkipTestExecutionFunctionalTest {
+
+    @ClassRule
+    public static final GitClone GIT_CLONE = new GitClone();
+
+    @Rule
+    public TestBed testBed = new TestBed(GIT_CLONE);
+
+    private static final String EXPECTED_LOG_PART = "Enabling Smart Testing";
+    private ByteArrayOutputStream logCapturingStream;
+    private PrintStream oldPrintStream;
+
+    @Before
+    public void attachLogCaterer() {
+        logCapturingStream = new ByteArrayOutputStream();
+        PrintStream newPrintStream = new PrintStream(logCapturingStream);
+        oldPrintStream = System.out;
+        System.setOut(newPrintStream);
```

Comment:
`BuiltProject` from Embedded Maven we are using under the hood in `ProjectBuilder` exposes `getMavenLog()`. So I was thinking - maybe instead of replacing the output stream to ours for capturing how about making this log available through DSL? This would also let us run those tests not sequentially.